andrepereiradasilva/joomla-cms
# Use the new useglobal form attribute in editors fields

Pull Request for Improvement.

### Summary of Changes

This PR makes use of the new use global on the editors fields in com_users (site/administrator profile and new/edit user).

### Testing Instructions

1. Use latest staging, apply patch #xxxxxxx (if not merged) and them apply this patch
2. Confirm with/without editors plugins enabled the options still work and properly inherit the setting from global config.
3. Confirm when editors plugin is selected and enabled, is still displayed for the user in the several scenarios

#### Before
xxxxx

#### After
xxxxxxxx

### Documentation Changes Required

None.
